gtk/rgdebinstallprogress.cc

```diff
@@ -510,7 +510,7 @@ void RGDebInstallProgress::terminalAction(GtkWidget *terminal,
         TermAction action) {
    switch(action) {
       case EDIT_COPY:
-          vte_terminal_copy_clipboard(VTE_TERMINAL(terminal));
+          vte_terminal_copy_clipboard_format(VTE_TERMINAL(terminal), VTE_FORMAT_TEXT);
```

Comment:
I just had a quick look and it seems vte_terminal_copy_clipboard_format() was added in 0.5 and it seems in xenial we have 0.42.5. Is that what you mean that for getting fully warning free we would stop building on xenial?

Reply:
Actually I was referring to the gtk warnings that have nothing to do with this commit.. oops..  except they also produce warnings namely gtk-menu-popup is deprecated - https://developer.gnome.org/gtk3/stable/GtkMenu.html#gtk-menu-popup

Reply:
Thanks for your comments. Let's merge it but I think we need to update configure.ac to have something like  
```
vte_modules="vte-2.91 >= 0.5"
```
is that correct?

Reply:
You are indeed correct.